### DISCUSSION TITLE
Rendering Headers with ANSI Colour Escapes?

### DISCUSSION THREAD

**Discussion by bennypowers** [goolord/alpha-nvim#89] [Q&A]
Hey, can I render a header that uses ANSI colour escapes? I want to do some pixel art

**Comment by goolord** [comment]
you options are

1. 2d array of highlight groups https://github.com/goolord/alpha-nvim/blob/main/doc/alpha.txt#L111
2. wait for https://github.com/goolord/alpha-nvim/pull/63

**Reply by bennypowers** [reply]
would this help? https://www.vim.org/scripts/script.php?script_id=302

**Reply by goolord** [reply]
maybe! try it out and lmk. seems cool
